### PR TITLE
Update support checklist

### PR DESCRIPTION
**Pull request by cnelson**
This updates the support checklist with the items mentioned in [18F-3312](https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3132)

I've removed the requirement to manually review CloudWatch Logs until we can do something about the issues raised in [18F-3637](https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3637) as until we do it's a sisyphean task that's unlikely to produce any useful findings.

### CODE REVIEW

**Comment by LinuxBozo** [comment]
I found I was doing one more thing: Reviewing new ZenDesk tickets and providing internal notes as necessary so that Tier 1 support could formulate an answer for more technical questions. Not sure if that's best practice though, and we should wait for Tier 1 to assign/ask for assistance.

**Comment by cnelson** [comment]
@brittag ☝️ Appreciate your opinion on @LinuxBozo's comment.  Is that helpful and something we should formalize by putting into the support checklist, or is it enough to wait for someone to 🖐 for support help?

File: content/docs/ops/maintenance-list.md [hunk 1]
```diff
@@ -5,29 +5,69 @@ menu:
...
-* Nessus security scanning reports
-* [Upstream stemcell releases](http://bosh.cloudfoundry.org/stemcells/) for security patches
-* [US Cert Alerts](https://www.us-cert.gov/ncas/alerts)
-* [Pivotal Alerts](https://pivotal.io/security)
+The platform requires regular support and maintenance activities to remain in a compliant state. If you are on support and you can’t complete any of the items personally, you are responsible for ensuring that an appropriate person does it.
+
+# Is it your first day of support?
+
+- Update the `#cloud-gov-atlas` topic to include your name as the support contact.
+- Update the support schedule by moving yourself to the end of the list.
```
Comment (LinuxBozo) [inline content/docs/ops/maintenance-list.md]: WDYT about expanding this to include it's also in `#cloud-gov-atlas` and the reminder syntax?

Comment (cnelson) [reply content/docs/ops/maintenance-list.md]: Updated, and edited the reminder in slack to point to this checklist.

File: content/docs/ops/maintenance-list.md [hunk 2]
```diff
@@ -5,29 +5,69 @@ menu:
...
+
+### Is the alert a real issue?
+Remediate it.
+
+### Is the alert a false-positive?
+If the alert can be tuned to to reduce the number of false-positives with less than one day's work, do it.  If more work is required to tune the alert, add a card to capture the work that needs to be done or +1 an existing card if one already exists for tuning the alert.
+
+Be prepared to represent support needs at the next grooming meeting to ensure that cards to fix alerts are prioritized properly.
+
+## Review AWS CloudTrail logs for the following Events:
```
Comment (jmcarp) [inline content/docs/ops/maintenance-list.md]: Is it worth writing a script for this (list all events of the relevant types in the last day not run by the terraform user), or is the console good enough? Or should we (eventually) write an alarm to notify us when this happens?

Edit: this is obviously out of scope for this PR, just hoping to capture more stories if necessary.

Comment (cnelson) [reply content/docs/ops/maintenance-list.md]: I've been doing this every day, and it takes less than 10 minutes via the console, so I don't think it's worth automating yet, but as the goal is to automate everything, I'll make a card for it as there are numerous approaches we could take and it's probably worth some discussion before we start writing code.

Comment (cnelson) [reply content/docs/ops/maintenance-list.md]: https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3639

**Comment by brittag** [comment]
@LinuxBozo @cnelson Reviewing support requests and writing diagnosis notes is very helpful!

The one issue though is that I don't see any form of notification when people write "internal notes" in Zendesk - I only see those notes when I'm looking directly at the support mails. It looks like [Zendesk only supports notifications for this kind of thing in a cumbersome way](https://support.zendesk.com/hc/en-us/community/posts/206052578-Internal-Note-Notification). This means I might not notice when a person writes a helpful note on an older message that I haven't looked at recently.

So it might be even better to recommend writing notes as comments in cg-supportstream, if that works well for you all?

**Comment by cnelson** [comment]
@LinuxBozo @brittag 

Added a section about responding to zendesk tickets.  

WDYT?

File: content/docs/ops/maintenance-list.md [hunk 3]
```diff
@@ -5,29 +5,76 @@ menu:
...
+
+### Is a stemcell update required to remediate?
+Ask for a date when new stemcells will be delivered in #security in the [CF Slack](https://cloudfoundry.slack.com/).
+
+### Is a bosh release update required to remediate?
+Update the bosh release and file a PR for the changes.  Once the PR is merged, ensure the updated release is deployed to all required VMs.
+
+## Review open support requests
+
+Review open Zendesk tickets and provide technical support / details as needed.  When possible reply directly to the customer; if the customer response requires internal discussion post any relevant information in `#cg-supportstream` and coordinate a response with the rest of the team.
```
Comment (brittag) [inline content/docs/ops/maintenance-list.md]: I'd suggest something like this:

> Review the "new" (yellow) and "open" (red) Zendesk tickets. First-tier support (cloud.gov BU) has primary responsibility to do the work of answering these, and you're second-tier support with technical expertise. You're welcome to reply to the customer with answers if you like (choose "pending" when you submit the answer),* but your main task is just to help with technical diagnoses/advice/details (especially in areas first-tier support isn't familiar with). The easiest way to do that is to write comments on the associated posts in `#cg-supportstream`. First-tier support may also ask you for pairing time to work out responses together.

> *People with @gsa.gov emails can't receive email via Zendesk, so we have to email them via the [cloud-gov-support Google Group instead](https://groups.google.com/a/gsa.gov/forum/#!forum/cloud-gov-support).

**Review by LinuxBozo — APPROVED** [review]
LGTM

File: content/docs/ops/maintenance-list.md [hunk 4]
```diff
@@ -5,29 +5,78 @@ menu:
...
+Review all recent alerts and notifications delivered to `cg-notifications` and `#cloud-gov-atlas-news`.  
+
+### Are there no alerts or notifications?
+Verify the monitoring system is functioning correctly and confirm that alerts are reaching their expected destinations.
+
+### Is the alert a real issue?
+Remediate it.
+
+### Is the alert a false-positive?
+If the alert can be tuned to to reduce the number of false-positives with less than one day's work, do it.  If more work is required to tune the alert, add a card to capture the work that needs to be done or +1 an existing card if one already exists for tuning the alert.
```
Comment (mogul) [inline content/docs/ops/maintenance-list.md]: s/to to/to/

File: content/docs/ops/maintenance-list.md [hunk 5]
```diff
@@ -5,29 +5,78 @@ menu:
...
+- PutUserPolicy
+- PutRolePolicy
+- RevokeSecurityGroupEgress
+- RevokeSecurityGroupIngress
+- UpdateTrail
+
+If any suspicious activity is discovered discuss with appropriate team members and if neccessary follow the [Security Incident Response Guide]({{< relref "docs/ops/security-ir.md" >}}).
+
+## Review vulnerability and compliance reports
+- Nessus Vulnerability Reports
```
Comment (mogul) [inline content/docs/ops/maintenance-list.md]: Should be explicit about how/where to find these.

File: content/docs/ops/maintenance-list.md [hunk 6]
```diff
@@ -5,29 +5,78 @@ menu:
...
+- Nessus Compliance Reports
+- [CloudFoundry Alerts](https://www.cloudfoundry.org/category/security/)
+- [US Cert Alerts](https://www.us-cert.gov/ncas/alerts)
+
+If the reports contain any HIGH items work to remediate them.
+
+### Is an update from our IaaS provided required to remediate?
```
Comment (mogul) [inline content/docs/ops/maintenance-list.md]: s/provided/provider/

File: content/docs/ops/maintenance-list.md [hunk 7]
```diff
@@ -5,29 +5,78 @@ menu:
...
+
+### Is a stemcell update required to remediate?
+Ask for a date when new stemcells will be delivered in #security in the [CF Slack](https://cloudfoundry.slack.com/).
+
+### Is a bosh release update required to remediate?
+Update the bosh release and file a PR for the changes.  Once the PR is merged, ensure the updated release is deployed to all required VMs.
+
+## Review open support requests
+
+Review the "new" (yellow) and "open" (red) Zendesk tickets. First-tier support (cloud.gov BU) has primary responsibility to do the work of answering these, and you serve as second-tier support providing technical expertise. You're welcome to reply to the customer with answers if you like (choose "pending" when you submit the answer),* but your main responsibility is to provide technical diagnoses/advice/details. The easiest way to do that is to write comments on the associated posts in `#cg-supportstream`. First-tier support may also ask you for pairing time to work out responses together.
```
Comment (mogul) [inline content/docs/ops/maintenance-list.md]: s/,\*/\*,/

**Comment by mogul** [comment]
Modulo these few typos, LGTM!

**Comment by cnelson** [comment]
Updated with additional links to documentation and nessus.

**Comment by mogul** [comment]
Still [WIP]?

**Review by rogeruiz — APPROVED** [review]
These are great. I will merge this in. @cnelson did you turn off the emails of Nessus Scans? I can do it as part of the acceptance criteria too. NBD.

**Comment by cnelson** [comment]
I disabled the nessus SMTP configuration entirely, but left the notification preferences in place on existing reports in case we need to turn it back on at some point.

You can verify this by looking at the SMTP configuration for nessus in the UI, and you shouldn't have any emails from nessus GovCloud after 3/12 (I made the change on 3/13)

**Comment by rogeruiz** [comment]
yeah just verified it myself. love the subdomain. 👏

**Pull request merged**